uqcsbot/join.py

```diff
@@ -0,0 +1,103 @@
+import discord
+from discord.ext import commands
+
+from uqcsbot.bot import UQCSBot
+from uqcsbot.models import Channel
+
+JOINED_PERMISSIONS = discord.Permissions(read_messages=True)
+SERVER_ID = 813324385179271168
+# Testing Server
+# SERVER_ID = 836589565237264415
+
+EMOJIS = {"academic-advice": "📖", "adulting": "🧑", "banter": "💬", "bot-testing" : "🤖",
+                "contests" : "⚔️", "covid" : "⚛️", "creative" : "🎨", "events" : "🗓️", "food" : "🍔",
+                "games" : "🎮", "general" : "⚪", "hackathons" : "🍕", "hardware" : "💻", "jobs-bulletin" : "📌",
+                "jobs-discussion" : "🔈", "lgbtqia" : "🏳️‍🌈", "media" : "📺", "memes" : "🎭", "politics" : "📮",
+                "projects" : "🔨", "uqic-sport" : "🏅", "yelling" : "🗣️"}
```

Comment:
Building upon what we do already have, would it be beneficial to store these alongside the channels within the database already? Also means we don't have to constantly update the code if we add/remove channels from the Discord.

Reply:
Thanks for the review. That is definitely a much better long-term solution. I will look at updated the db to store these values instead of inside the code itself